api/integrations/rudderstack/rudderstack.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import typing
+
+import rudder_analytics
+
+from integrations.common.wrapper import AbstractBaseIdentityIntegrationWrapper
+
+if typing.TYPE_CHECKING:
+    from environments.identities.models import Identity
+    from features.models import FeatureState
+
+logger = logging.getLogger(__name__)
+
+
+class RudderstackWrapper(AbstractBaseIdentityIntegrationWrapper):
+    def __init__(self, base_url: str, api_key: str):
+        rudder_analytics.write_key = api_key
+        rudder_analytics.data_plane_url = base_url
+
+    def _identify_user(self, user_data: dict) -> None:
+        rudder_analytics.identify(user_data.get("user_id"), user_data.get("traits"))
```

Comment:
Looking at the `generate_user_data` method, there's no reason that this can't be: 

```python
rudder_analytics.identify(**user_data)
```